collections/_article/progress-report-state-of-particles.md

```diff
@@ -0,0 +1,94 @@
+
+### Changes in `ParticleProcessMaterial`
+
+The main change in the particle material is the big-big refactor of the internal code. The code was split into smaller, reusable functions, which are
+called both from `start()` and `process()`. This ensures better consistency and maintainability.
+
+Multiple configuration options were added or changed to expand control over damping, emission, and velocity.
+
+Damping changes include:
+
+- `damping_as_friction`: flag that changes the behavior of damping from a constant
+deceleration to a deceleration based on speed.
+
+Emission changes include:
+
+- `emission_shape_offset`: an offset for the emission shape, in the node's local space.
+- `emission_shape_scale`: the scale of the emission shape, in the node's local space.
+- `inherit_velocity_ratio`: a percentage of the emitter's velocity to inherit on spawn.
+
+Velocity changes include:
+
+- `velocity_pivot`: a pivot point used for calculating radial and orbital velocity.
+- `directional_velocity`: velocity along an axis; requires an XYZ curve.
+- `orbit_velocity`: orbit velocity can now be used without toggling `disable_z`; requires an XYZ curve; axes are in the node's local space.
+- `radial_velocity`: velocity away or towards the velocity pivot.
+- `velocity_limit`: a curve defining a hard limit for the particles' velocity following their lifetime.
+- `scale_over_velocity`: a curve that allows to scale particles on a given axis.
+
+On top of that an optional separation between alpha over lifetime and emission over lifetime was added. Note that emission over lifetime just multiplies the `COLOR` value of the particle by a given value. This needs to be properly read in the display shader.
+
+You can also check out [GH-79527](https://github.com/godotengine/godot/pull/79527), which covers the bulk of these changes.
+
+I can't wait to see all that you'll make with it! In the meantime, enjoy these VFX made by the testers of the PR.
+
+## And what about `CPUParticles`?
+
+CPU particles have been maintained alongside GPU particles for a long time, but it's becoming [harder and harder to keep feature parity](https://github.com/godotengine/godot-proposals/issues/7344), especially after big changes such as the ones described above. CPU particles were added at the time of the GLES2 renderer in Godot 3, which did not have the capability to support GPU particles. Godot 4 has Vulkan and GLES3 renderers, both of which support GPU particles. This makes it hard to justify maintaining four separate nodes in two different languages (GLSL and C++).
+
+Unfortunately, unless there are people willing to maintain `CPUParticles`, they will begin to diverge further from `GPUParticles` in future releases. It's likely that no more features will be added to `CPUParticles`. We recommend migrating your effects to `GPUParticles`. To help you with the migration, [a converter from CPU particles to a GPU equivalent](https://github.com/godotengine/godot/pull/80779) was added.
```

Comment:
```suggestion
Accordingly, we now see the `CPUParticles` as a lower-end, simple alternative to GPUParticles, instead of a CPU-based, 1-to-1 fallback. It's likely that no more features will be added to `CPUParticles`. We recommend migrating your effects to `GPUParticles`. To help you with the migration, [a converter from CPU particles to a GPU equivalent](https://github.com/godotengine/godot/pull/80779) was added.
```

This sounds like it is blaming the reader for not doing more. I don't think there is any need for this ultimatum anyway

Reply:
I agree, good suggestion! I couldn't figure out what to do with this sentence myself.

Reply:
I adjusted the new message to mention welcoming PRs for parity; and also to make it sound more like Qbie :)